pkg/server/server.go

```diff
@@ -549,6 +603,16 @@ func NewServerWithOpts(opts ...OpenFGAServiceV1Option) (*Server, error) {
 		s.listObjectsDispatchThrottler = throttler.NewConstantRateThrottler(s.listObjectsDispatchThrottlingFrequency, "list_objects_dispatch_throttle")
 	}
 
+	if s.listUsersDispatchThrottlingEnabled {
+		s.logger.Info("Enabling ListUsers dispatch throttling",
```

Comment:
nit, you don't need this log, it should be logged automatically on server startup (https://github.com/openfga/openfga/pull/1609)

Reply:
Yes, this would just add noise. Please drop.

Reply:
Ah wait, there is one caveat, the line that prints all the configs (https://github.com/openfga/openfga/blob/f10d8e5df8e3d0ecfcd91a4e1a772dab32d8d360/cmd/run/run.go#L619-L625) doesn't run if you import OpenFGA as a library 😭